mappings/net/minecraft/client/util/telemetry/TelemetryManager.mapping

```diff
@@ -0,0 +1,44 @@
+CLASS net/minecraft/unmapped/C_wwbyxkgc net/minecraft/client/util/telemetry/TelemetryManager
+	FIELD f_dgegipht session Lcom/mojang/authlib/minecraft/TelemetrySession;
+	FIELD f_dkujvsyq client Lnet/minecraft/unmapped/C_diboefya;
+	FIELD f_egikwkbj TELEMETRY_SENDER_COUNTER Ljava/util/concurrent/atomic/AtomicInteger;
```

Comment:
What does this do exactly? Feels like there's definitely a better name for it.

Reply:
It counts the number of threads in `TELEMETRY_SENDER_EXECUTOR`. Any suggestions for a name?

Reply:
While the field is `static final`, i feel like Atomic values have different usecases than actual `static final` fields. I would consider naming this `telemetrySenderThreads`